firmware/dev/f4dev/src/app/configs/App_HeartbeatMonitorConfig.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#define HEARTBEAT_MONITOR_TIMEOUT_PERIOD_MS 300U
+#define HEARTBEAT_MONITOR_BOARDS_TO_CHECK (enum HeartbeatOneHot)(BMS_HEARTBEAT_ONE_HOT)
```

Comment:
You should probably actually define this as `0U`. Or even better would be to omit the shared heartbeat driver from compilation and you don't need this config header.

Reply:
I copy this from `DIM` because I can't build without the config. But now it can build without this config. Now it is removed.